### DISCUSSION TITLE
Topics for the "Discussions" section

### DISCUSSION THREAD

**Discussion by flxzt** [flxzt/rnote#636] [Announcements]
The "Discussions" section is supposed to be used for announcements, questions and open ended discussions.

If you have a specific feature request or are experiencing a bug, please open a new issue by navigating into the "Issues" section and clicking on "New Issue" or directly by clicking [here](https://github.com/flxzt/rnote/issues/new/choose).

Thanks!

**Comment by rubiojr** [comment]
:wave: @flxzt, rnote is mind blowing :exploding_head:, thanks for it!

Any chance you have plans to test and develop this in a new PineTab2? What devices would you recommend for it?

**Reply by flxzt** [reply]
Thanks :) Though I am glad that I can say that my regular contributors also deserve some of the credit by now.

Running the app on aarch64 devices like the pine tab 2 should already work (through the flathub flatpak), though I am not sure if it is powerful and the display good enough that it will be a good experience. I am more excited for devices with ePD like the PineNote, I want to get my hands on one for a while now and improve support for that.

I can only speak from my personal experience: The x1-yoga works really well for me, in general wacom seems to be the best supported vendor for styli on linux. Some users reported issues with HP-Pens (#194), so maybe it is best to avoid them. Maybe also check discussion #77 to hear what other users report.
